app/modules.go

```diff
@@ -245,6 +249,8 @@ func orderBeginBlockers() []string {
 		wasmtypes.ModuleName,
 		ibcwasmtypes.ModuleName,
 		tftypes.ModuleName,
+		marketmaptypes.ModuleName,
```

Comment:
Wondering if this is the order here? As slinky needs to prioritised then it should be on the top of `begin_blockers`, `end_blockers` and `init_blockers`

Reply:
so slinky process uses vote extensions to resolve marketmap feeds - the market map module just has the state so matter in ordering